content/builds/power-deadeye.md

```diff
@@ -0,0 +1,88 @@
++++
+title = "Power Unload Deadeye"
+description = "This build is designed around the skill Unload, being able to refresh Relic of the Thief and being self sufficient in initiative use."
+date = 2024-03-29
+draft = false
+template = "build.html"
+
+[taxonomies]
+tags = ["power","thief","deadeye","pof","lowcog","lowphys","lowrep"]
+authors = ["xellink"]
+
+[extra]
+series = "thief"
+tagline = "Auto-Attack Deadeye with/without Quickness"
+keywords = "Guild Wars 2, GW2, LI, Deadeye"
+toc = true
+spec = "deadeye"
+balance = "March 2024"
++++
+
+The purpose of writing this guide is to provide a very low intensity way to play PvE deadeye. This build is low cognition, low physical complexity, has very few repetitions and is functional in both instanced content. There is an optional amount of APM to push the build to its limit. While this build has a high skill floor, it has a low skill ceiling, requiring daggers and a few button presses to push its damage to near competitive levels.
+At the time of writing, there options for deadeye along the power DPS options does not smoothly transition players from low to high intensity. This simplified build also allows you to perform niche roles like pylon kiting, which is extremely useful when learning the role and not wanting to be bogged down by rotation.
+I have completed all 7 wings in NM with this build being fairly high/average ranked in DPS meters. I do not think that bringing this build is going to gimp your team, but if there are no limitations, there are better options that I hope you would consider as an alternate set to complement this build when you are ready starting with dual daggers. Refer to Section D1.
+The improvised quickness variant uses the same stats and is able to overcap on quickness through a very simple rotation and minimal use of cantrips.
+
+## Gearing
+
+- Full Berserker's up to the crit cap. It is okay to overcap on precision due to the occasional need to switch to the practiced tolerance trait.
+- Dragon’s/Valkyrie+Berserker if desired as the build is heavily overcapped on precision
+- Runes of the Scholar/Dragonhunter
+- Dual Pistols/Dual Daggers
+- Relic of the Thief
+- Consumables:
+  - Cilantro Lime Sous-Vide Steak (Power/Ferocity)
+  - Moa Wings is a suitable budget alternative
+  - Superior Sharpening Stone (Power)
+
+## Build
+DPS
+{{ chatlink(code="[&DQUjHxw9Oi+FAIUAMwFYAVgBMwGsFlYBKAAoAAAAAAAAAAAAAAAAAAAAAAACNgAvAAA=]") }}
+
+Quickness Variant
+{{ chatlink(code="[&DQUjGxwtOj+FAAwBMwFYASAXMwFYAFYBDgEoAAAAAAAAAAAAAAAAAAAAAAACNgAvAAA=]") }}
+
+## Trait Alternatives
+1. Practiced Tolerance - For fights where power cleave/pierce is detrimental, you should also take {{ trait(id="1272") }} (eg. Soulless Horror). You can freely switch to daggers if you take this trait.
```

Comment:
trait() shortlink isn't implemented yet, so this will not build. Currently skill() only